src/test/java/Homework18.java

```diff
@@ -0,0 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+
+    @Test
+    public void playSong() throws InterruptedException {
+        navigateToPage();
+        login("jenyag5@gmail.com", "te$t$tudent");
+        Thread.sleep(2000);
+        allSongs();
+        songName();
+        playNextButton();
+        playButton();
+        soundBars();
+
+    }
+    public void allSongs() throws InterruptedException {
+        WebElement allSongs = driver.findElement(By.cssSelector("#sidebar > section.music > ul > li:nth-child(3) > a"));
+        allSongs.click();
+        Thread.sleep(2000);
+    }
+    public void songName() throws InterruptedException {
+        WebElement songName = driver.findElement(By.xpath("//*[@id=\"songsWrapper\"]/div/div/div[1]/table/tr[1]/td[2]"));
+        songName.click();
+        Thread.sleep(2000);
+    }
+    public void playNextButton() throws InterruptedException {
+        WebElement playNextButton = driver.findElement(By.xpath("//i[@data-testid='play-next-btn']"));
+        playNextButton.click();
+        Thread.sleep(2000);
+    }
+    public void playButton() throws InterruptedException {
+        WebElement playButton = driver.findElement(By.xpath("//span[@data-testid='play-btn']"));
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210991939-0e3b495d-8a08-495f-b4f4-15cd6094ece3.png)

![image](https://user-images.githubusercontent.com/104761482/210992195-fbc78526-b093-4247-b5da-6dac40b3f004.png)

Hint

Selenium Actions allow you to perform UI actions in your test.
Clicking, double-clicking, hovering or other complex mouse actions can be scripted with an action.

![image](https://user-images.githubusercontent.com/104761482/210992455-728c39ca-fac9-4472-bb4b-d6a10bcd76df.png)